test/e2e/apimachinery/discovery.go

```diff
@@ -142,8 +142,8 @@ var _ = SIGDescribe("Discovery", func() {
 			checkGroup := &metav1.APIGroup{}
 			apiPath := "/apis/" + group.Name + "/"
 			err = f.ClientSet.Discovery().RESTClient().Get().AbsPath(apiPath).Do(ctx).Into(checkGroup)
-			framework.ExpectNoError(err, "Fail to access: %s", apiPath)
-			framework.ExpectNotEqual(len(checkGroup.Versions), 0, "No version found for %v", group.Name)
+			framework.ExpectNoError(err, "Fail to access: %s", apiPath))
```

Comment:
```suggestion
			framework.ExpectNoError(err, "Fail to access: %s", apiPath)
```
There is a stray ')' at the end of the line. Please fix it and I'd be happy to lgtm again :+1: